examples/suit_update/README.md

```diff
@@ -84,39 +103,53 @@ Once everything is configured you will get:
 
 Keep this running (don't close the shell).
 
-Add a routable address to host:
+From another terminal, add a routable address to host:
 
     $ sudo ip address add 2001:db8::1/128 dev tap0
 
-If the network has been started as described above, the RIOT node will be
-reachable via link-local "fe80::2" on the ethos interface and not by its wireless
-interface global address.
+If the network has been started as described above, the RIOT updatable device
+is reachable on its link-local address `fe80::2` via the `tap0` interface but
+not when using its wireless interface global address:
+
+    $ ping6 fe80::2%tap0
 
-## Wireless Node with Border Router
+### Setup a wireless device behind a border router
+[setup-wireless]: #Setup-a-wireless-device-behind-a-border-router
 
-### Setup network
+#### Configure the wireless network
+[setup-wireless-network]: #Configure-the-wireless-network
 
-A wireless node has no direct connection to the Internet we need to setup a border
-router, to this use any node with a 802.15.4 radio and follow the instructions to
-setup [gnrc_border_router](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_border_router).
-Pay attention to the prefix you will use, if you follow the border router instructions
-it would be '2001:db8::/64' when running:
+A wireless node has no direct connection to the Internet so a border router
+between 802.15.4 and Ethernet must be configured.
+Any board providing a 802.15.4 radio can be used as a border router.
 
-    $ sudo sh start_network.sh /dev/ttyACMx tap0 2001:db8::1/64
+Plug the board that will act as a border router on the computer and flash
+the
+[gnrc_border_router](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_border_router)
+application on it:
 
-Add a routable address to host:
+    $ make BOARD=<br board> -C examples/gnrc_border_router flash
```

Comment:
$ make BOARD=<board> -C examples/gnrc_border_router flash

Reply:
?

Reply:
ups sorry I meant: `$ make BOARD=<board> -C examples/gnrc_border_router flash` or is the `br board` on purpose?

Reply:
Yes br stands for border router. Because it might another type of board than the board running the updatable firmware.

Reply:
ahhhh ok! Maybe use caps since it's the notation used in [gnrc_border_router](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_border_router)?